resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnAllocatorBlacklistTrackerSuite.scala

```diff
@@ -137,4 +144,41 @@ class YarnAllocatorBlacklistTrackerSuite extends SparkFunSuite with Matchers
     verify(amClientMock).updateBlacklist(Arrays.asList("host4"), Collections.emptyList())
     assert(yarnBlacklistTracker.isAllNodeBlacklisted === true)
   }
+
+  test("test with initial blacklisting") {
+    sparkConf.set(YARN_EXECUTOR_INITIAL_BLACKLISTED_NODES, Seq("initial1", "initial2"))
+    val yarnBlacklistTracker = createYarnAllocatorBlacklistTracker(sparkConf)
+    yarnBlacklistTracker.setSchedulerBlacklistedNodes(Set())
```

Comment:
Just thinking aloud -- could this just be added to one of the other tests, eg. "combining scheduler and allocation blacklist"?  After the first check, the rest of this test doesn't seem distinct from the others, I'm not sure its worth having a separate test for it (harder to maintain redundant tests down the road if behavior changes, etc.)

Reply:
Sounds fine for me.